app/scripts/modules/core/src/utils/unicodeBase64.ts

```diff
@@ -0,0 +1,7 @@
+export const decodeUnicodeBase64 = (encodedString: string) =>
+  decodeURIComponent(
+    atob(encodedString)
+      .split('')
+      .map(char => '%' + ('00' + char.charCodeAt(0).toString(16)).slice(-2))
+      .join(''),
+  );
```

Comment:
turns out `atob` doesn't really know what to do with unicode chars, this is just copy-pasta that properly handles them.

Reply:
Worth putting a comment in the code about this, or pulling the function out into a nicely named function. None of us will remember what this is doing in a week.

Reply:
eh, i originally thought about a comment and decided against it because i thought the name was clear enough on its own 😃

Will throw in a quick comment.

Reply:
@jrsquared 2934804

Reply:
Noice.